doc/cephfs/administration.rst

```diff
@@ -49,7 +49,7 @@ Settings
 
     fs set <fs name> max_file_size <size in bytes>
 
-CephFS has a configurable maximum file size, and it's 1TB by default.
+CephFS has a configurable maximum file size, and it is 1TB by default.
```

Comment:
i am an ESL, and i am pretty fine with "it's" or "it doesn't". just out of curiosity, is there any research result showing that contractions is lower the readability of the document from an ESL's perspective?